app/scripts/related-entity-transform.js

```diff
@@ -2,9 +2,9 @@
  * transform elastic search related entity query to display format.  See data-model.json
  */
 
-/* globals _ */
+/* globals _, commonTransforms */
```

Comment:
Please add dateFormat to the list of globals here and as a parameter to the transform function on line 7.